libass/ass_utils.h

```diff
@@ -180,26 +181,12 @@ static inline int double_to_d22(double x)
     return lrint(x * 0x400000);
 }
 
-#define ASS_HASH_INIT 0x811c9dc5U
-#define FNV1_32A_PRIME 16777619U
+#define ASS_HASH_INIT 0U
 
-static inline uint32_t ass_hash_buf(const void *buf, size_t len, uint32_t hval)
+static inline uint64_t ass_hash_buf(const void *buf, size_t len, uint64_t hval)
 {
-    if (!len)
-        return hval;
-
-    const uint8_t *bp = buf;
-    size_t n = (len + 3) / 4;
-
-    switch (len % 4) {
-    case 0: do { hval ^= *bp++; hval *= FNV1_32A_PRIME; //-fallthrough
-    case 3:      hval ^= *bp++; hval *= FNV1_32A_PRIME; //-fallthrough
-    case 2:      hval ^= *bp++; hval *= FNV1_32A_PRIME; //-fallthrough
-    case 1:      hval ^= *bp++; hval *= FNV1_32A_PRIME;
-               } while (--n > 0);
-    }
-
-    return hval;
+    static const uint64_t _wyp[5] = {0xa0761d6478bd642full, 0xe7037ed1a0b428dbull, 0x8ebc6af09c88c6e3ull, 0x589965cc75374cc3ull, 0x1d8e4e27c47d124full};
```

Comment:
This exact value is already defined in the wyhash-header. Unless we decide to use a (possibly run-time generated) custom secret, this line can just be removed.